sdk/python/kfp/onprem.py

```diff
@@ -19,10 +21,55 @@ def _mount_pvc(task):
         return (
             task
                 .add_volume(
-                    k8s_client.V1Volume(name=volume_name, persistent_volume_claim=local_pvc)
-                )
+                k8s_client.V1Volume(name=volume_name, persistent_volume_claim=local_pvc)
```

Comment:
Linter is acting weird here...Do you mind fixing this?

Reply:
Pushed fix. Better?